src/app/shared/case-editor/case-edit-submit.html

```diff
@@ -28,13 +33,18 @@ <h2 class="plain-message">Check the information below carefully. This will form
             <ng-container *ngIf="canShowFieldInCYA(field)">
               <tr ccdConditionalShow ccdLabelSubstitutor [caseField]="field"
                 [formGroup]="editForm.controls['data']" [eventFields]="eventTrigger.case_fields">
-                <th class="valign-top">{{field.label}}</th>
+                <th class="valign-top"><span class="text-16">{{field.label}}</span></th>
                 <td class="form-cell">
-                  <ccd-field-read [caseField]="summaryCaseField(field)" [context]="paletteContext"></ccd-field-read>
+                  <a href="mailto:{{summaryCaseField(field).value}}" class="text-16" *ngIf="isEmail(field)">
```

Comment:
this should be changed. It's not responsibility of case-edit-submit.html to check whether a field has an email and display that accordingly. 
This should be moved inside the email read component template

Reply:
I tried but it was bit challenging due to that all types of read only fields are rendering dynamically.  May be I will look at that to improve sometimes in future if I find a better solution.

Reply:
becasuse theis field is used only on the check your answers section of this page, we should rename it to ccd-cya-field-read 
and we should move all this logic inside it's template. We should also move the isEmail method inside the ccdCYAFieldRead component
Karamath agree that he's going to open a new ticket to do this change